From 7eda24e5993c41c11733b67f3d74b6a3b9e10d4d Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Wed, 14 Oct 2020 22:49:14 +0900
Subject: [PATCH] do not install setup.sh

---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 270b98d..f153ef2 100644
--- a/setup.py
+++ b/setup.py
@@ -552,8 +552,7 @@ try:
                                       "sdist":OtherSetupForSdist, "install":Install, "install_data":InstallData },
                  packages         = unix_packages,
                  scripts= ['OpenRTM_aist/utils/rtcprof/rtcprof_python',
-                           'OpenRTM_aist/utils/rtcd/rtcd_python',
-                           'OpenRTM_aist/ext/sdo/observer/setup.sh'],
+                           'OpenRTM_aist/utils/rtcd/rtcd_python'],
                  data_files       = unix_data_files)
       
     # for RTM zip (sdist)
-- 
2.17.1

