From d16e9b12efc3cde254a401be7f4d408f59841751 Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Fri, 29 Sep 2023 16:54:22 +0900
Subject: [PATCH 4/4] 'string' does not support 'join'

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index db5d37d..0e399ed 100644
--- a/setup.py
+++ b/setup.py
@@ -47,7 +47,7 @@ def compile_idl(cmd, pars, files):
   Put together command line for python stubs generation.
   """
   global g_os
-  cmdline = cmd +' '+ string.join(pars) +' '+string.join(files)
+  cmdline = cmd +' '+ " ".join(pars) +' '+" ".join(files)
   if g_os == "win32":
     os.system(cmdline)
     return
-- 
2.25.1

